lightkurve/targetpixelfile.py

```diff
@@ -212,6 +211,11 @@ def find_stars(self, catalog=None, radius=0.5):
         radius: float
             Radius of cone search centered on the target in arcminutes.
             Default radius is 0.5 arcmin.
+        seperation_factor: float
```

Comment:
I still don't think separation factor should be a keyword...But I could be wrong. If we use it we should change this docstring, it's confusing. Let's use something like

Tolerance for source matching. Sources that fall within this tolerance of pixels in the KeplerTargetPixelFile will be returned in the result. Default is 3 pixels (12 arcsec).